src/value.rs

```diff
@@ -471,7 +472,13 @@ mod tests {
                 native_value_balance,
                 ValueCommitTrapdoor::zero(),
                 AssetId::native(),
-            ))
+            )
```

Comment:
please fix `bvk` to be
```rust
 let bvk = (values
            .into_iter()
            .map(|(value, rcv, asset)| ValueCommitment::derive(value, rcv, asset))
...
```
This is more elegant.

Reply:
definitely